html/semantics/document-metadata/the-link-element/link-type-attribute.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<link rel=match href=link-type-attribute-ref.html>
+<body>
```

Comment:
Nit: maybe omit the body tags as you did in the ref. A browser that fails might think that has something to do with it, while it's meaningless.